runtime/ops/os/mod.rs

```diff
@@ -386,6 +386,39 @@ fn rss() -> usize {
   task_info.resident_size as usize
 }
 
+#[cfg(target_os = "openbsd")]
+fn rss() -> usize {
+  let pid = unsafe { libc::getpid() };
+  let pagesize = unsafe { libc::sysconf(libc::_SC_PAGESIZE) } as usize;
+  let mut kinfoproc: std::mem::MaybeUninit<libc::kinfo_proc> =
+    std::mem::MaybeUninit::uninit();
+  let mut size = std::mem::size_of_val(&kinfoproc) as libc::size_t;
+  let mut mib = [
+    libc::CTL_KERN,
+    libc::KERN_PROC,
+    libc::KERN_PROC_PID,
+    pid,
+    size.try_into().unwrap(),
```

Comment:
**question:** What are we trying into here? This seems sort of odd to me since I have no idea about the types associated (except for `libc::size_t` which I'm not sure if it's the same as `size_t` or not :D )

Reply:
Added a comment explaining the rationale for this,